wagtail/core/management/commands/purge_revisions.py

```diff
@@ -0,0 +1,57 @@
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from wagtail.core.models import PageRevision
+
+try:
+    from wagtail.core.models import WorkflowState
+    workflow_support = True
+except ImportError:
+    workflow_support = False
+
+
+class Command(BaseCommand):
+    help = 'Delete page revisions which are not the latest revision for a page, published or scheduled to be published, or in moderation'
+
+    def add_arguments(self, parser):
+        parser.add_argument('--days', type=int, help="Only delete revisions older than this number of days")
+
+    def handle(self, *args, **options):
+        days = options.get('days')
+
+        revisions_deleted = purge_revisions(days=days)
+
+        if revisions_deleted:
+            self.stdout.write(self.style.SUCCESS('Successfully deleted %s revisions' % revisions_deleted))
+        else:
+            self.stdout.write("No revisions deleted")
+
+
+def purge_revisions(days=None):
+    # exclude revisions which have been submitted for moderation in the old system
+    purgable_revisions = PageRevision.objects.exclude(
```

Comment:
Sorry @jacobtoppm, I spelt 'purgeable' wrong in #5192 and you politely copied it here :) I think we should replace all `purgable`s with `purgeable`.

Reply:
Oops, thanks! I've fixed now.

Reply:
Thanks @jacobtoppm LGTM!